### ISSUE TITLE
Send armorstands via packets

### ISSUE THREAD

**Issue by EinHecki** [snowgears/shopbugs#92]
**Is your feature request related to a problem? Please describe.**
On big servers Shop can push the tps to a minimun if many players have many shops because of the big amount of real armor stands.

**Describe the solution you'd like**
I want to have the option to send the armorstand via packets. That would fix problems with armorstands which were not deleted, etc. The players wouldn't be affected of the better performance but for the server performance it would be much better. If this is too complicated to code for multiversion support I want at least support for 1.14+.

**Describe alternatives you've considered**
Well, I think there isn't really a alternative. 

**Additional context**
If you (the dev) want to make "Shop" the best and most efficient chest-shop plugin on spigot you really should think about implementing it because it would make life easier for users and owners.

**Comment by BillyGalbreath** [comment]
I got a better idea.

When sending fake armorstand packets, like you suggest, the plug-in has to keep track of them all and the players so it can keep resending the packets when players come in and out of viewing range. This not only is heavy task in itself, butnopens up the door for many bugs.

Rather than doing all that, what if we just set the armorstands to not tick using Paper's API? This way they are still real armorstands so the server keeps track of all the hard work, but they don't tick so they won't cause lag.

If you use Paper already, there is a config option to disable _all_ armorstands to not tick. I highly advise using that option unless ticking is required for your server.
